blockchain/utxocache.go

```diff
@@ -807,14 +814,15 @@ func (s *utxoCache) InitConsistentState(tip *blockNode, fastSync bool, interrupt
 
 		// We can update this after each batch to avoid having to redo the work
 		// when interrupted.
-		err := dbPutUtxoStateConsistency(dbTx, ucsConsistent, &node.hash)
-		return node, err
+		return node, dbPutUtxoStateConsistency(dbTx, ucsConsistent, &node.hash)
 	}
-	for node := statusNodeNext; node.height <= tip.height; {
-		log.Tracef("Replaying %d more blocks...", tip.height-node.height+1)
+
+	for node := statusNodeNext; node.height < tip.height; {
```

Comment:
I think we want to include the tip in the roll forward right?

Reply:
Yep we should. Fixed.